src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/managehearings/ManageHearingDocumentsCollectionItem.java

```diff
@@ -1,7 +1,22 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.managehearings;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
 import java.util.UUID;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Still need this?  Just checking.  I wouldn't expect any unexpected data on new structures like this.  So don't want to accidentally miss any new JSON fields added.